demo/src/components/RangeFilters.js

```diff
@@ -60,6 +60,7 @@ const RangeFilters = () => {
             FacetListItemComponent={FacetListItemComponent}
             collapsible={true}
             enableViewMore={true}
+            minViewMore={3}
```

Comment:
Please add `enableViewMore ` and  `minViewMore` on the main component propsTypes with description. In this case the component will be `RangeFacets.js` This will be picked up by storybook. [ref](https://github.com/unbxd/react-search-JS-SDK/blob/acfe794712d10ea4f9c1103d7d280410b8123740/src/modules/pagination/Pagination.js) 


Do this on all the components that take props `enableViewMore ` and  `minViewMore` .

Reply:
done